src/js/components/__tests__/ServicesTable-test.js

```diff
@@ -8,11 +8,11 @@ const React = require('react');
 /* eslint-enable no-unused-vars */
 const ReactDOM = require('react-dom');
 const ServicesTable = require('../ServicesTable');
-const Service = require('../../structs/Service');
+const Application = require('../../structs/Application');
```

Comment:
Not true, because if I instantiate a new `Service`, all the health checks will always return dummy data, since `Service` is an abstract class:

```js
  getTasksSummary() {
    return {
      tasksHealthy: 0,
      tasksStaged: 0,
      tasksUnhealthy: 0,
      tasksUnknown: 0,
      tasksOverCapacity: 0,
      tasksRunning: 0
    };
  }
```

I add a test for a `Pod` instance when the struct is ready.